opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py

```diff
@@ -143,6 +143,7 @@ def __init__(
         self.kind = kind
 
         self.span_processor = span_processor
+        self.status = trace_api.Status()
```

Comment:
We could add constant of type `Status` named `OK` for the `Status(OK, None)` object in the status module and use it here to save object allocations.

Reply:
Why adding a constant if this is already the default Status?, is this some kind of pattern? sound a little odd for me

Reply:
It's a small optimization to avoid creating a new "OK" status with each new span when they could all use the same instance instead.

OK statuses are all alike, every not-OK status is not-OK in its own way. So we may as well use a single instance for the OK status.